index.html

```diff
@@ -15,31 +15,316 @@
-                <b></b> I can help you build a product , feature or website Look through some of my work and experience! If you like <br/>what you see and have a project you need coded, don’t hestiate to contact me. </p>
-
-            <ul class="icon-group">
-                <li class="icon"> <a href=""><i class="fa-brands fa-twitter"></i></a></li>
-                <li class="icon"> <a href=""><i class="fa-brands fa-linkedin"></i></a></li>
-                <li class="icon"> <a href=""><i class="fa-solid fa-m"></i></a></li>
-                <li class="icon"> <a href="#https://github.com/TiteLunda"><i class="fa-brands fa-github"></i></a></li>
-                <li class="icon"> <a href=""><i class="fa-brands fa-angellist"></i></a></li>
-            </ul>
+            <nav class="menu">
+                <i class="fa-solid fa-bars "></i>
+            </nav>
+        </header>
+
+        <section class="header">
+            <div class="container">
+                <h1 class="identity">
+                    Hey there. I'm, <br/> Tite Lunda <br><span>I'm a software developer</span>
+                </h1>
+                <p class="profile">
+                    <b></b> I can help you build a product , feature or website Look through some of my work and experience! If you like <br/>what you see and have a project you need coded, don’t hestiate to contact me. </p>
+
+                <ul class="icon-group">
+                    <li class="icon"> <a href=""><i class="fa-brands fa-twitter"></i></a></li>
+                    <li class="icon"> <a href=""><i class="fa-brands fa-linkedin"></i></a></li>
+                    <li class="icon"> <a href=""><i class="fa-solid fa-m"></i></a></li>
+                    <li class="icon"> <a href="#https://github.com/TiteLunda"><i class="fa-brands fa-github"></i></a></li>
+                    <li class="icon"> <a href=""><i class="fa-brands fa-angellist"></i></a></li>
+                </ul>
+                <ul class="icon-group">
+                    <li><i class="fa-solid fa-chevron-down">
+                    </i>
+                    </li>
+                </ul>
+            </div>
+        </section>
+        <section class="works">
+            <div class="Recent">
+                <h2>My Recents Works</h2>
+            </div>
+            <div class="cards">
+                <img src="/Images/Img Plaholder.png" alt="Store image">
+                <h4 class="title">Multi-Post Stories Gain+Glory</h4>
+                <ul class="Languages">
+                    <li class="language"><a href="#">Ruby on rails</a></li>
+                    <li class="language"><a href="#">Css</a>
+                    </li>
+                    <li class="language">
+                        <a href="#">Javascript</a>
+                    </li>
+                    <li class="language"><a href="#">Html</a></li>
+                </ul>
+                <div class="b_div"><button class="button">See Project</button></div>
+            </div>
```

Comment:
[OPTIONAL] This is optional 🤷🏽 as it wasn't mentioned by the previous reviewer but make sure to adjust this design so that your design matches the Figma. 

## Figma
![Screenshot 2023-02-18 at 17 17 13](https://user-images.githubusercontent.com/45739963/219901722-04a13bac-ea96-4afc-a679-c9d6ff2bd2c2.png)


## Your design
![Screenshot 2023-02-18 at 17 15 33](https://user-images.githubusercontent.com/45739963/219901672-12c6c439-66e3-4311-af66-56eaa6a50720.png)